### ISSUE TITLE
Daily Notes Tagged Events

### ISSUE THREAD

**Issue by mrkuramoto** [davish/obsidian-full-calendar#284]
Regarding Daily Notes, I would like to request a simple TAG to mark events that go to  Full Calendar. Instead of "[allDay:: true]", a TAG like #fullcalendar... everything in the Daily Notes tagged with #fullcalendar appears on the calendar.

**Comment by AliAboelela123** [comment]
I would also really like a feature where just any note across my entire vault with a certain #tag would show up in my calendar so long as it has the necessary properties e.g date.

**Comment by ryguigas0** [comment]
> I would also really like a feature where just any note across my entire vault with a certain #tag would show up in my calendar so long as it has the necessary properties e.g date.

Yes that would help me a lot if I didnt need to restructure my folders to include the calendar tasks

**Comment by kittensmittens** [comment]
I would like to better sort my notes. Currently they are all just in the main directory of the vault, and when I move them into a folder they disappear from the calendar.
